apps/storybook-react/stories/Accordion/Accordion.docs.mdx

```diff
@@ -39,8 +41,7 @@ const {
 
 ```jsx
 import { Accordion } from '@equinor/eds-core-react'
-
-<Accordion>
+;<Accordion>
```

Comment:
```suggestion
<Accordion>
```

Reply:
It's simply not possible for me to save without I guess prettier adding that semicolon :/